### PR TITLE
Add support for // in ybt_bin_path

### PR DESCRIPTION
**Pull request by eyal-resonai**

### CODE REVIEW

File: yabt/docker.py [hunk 1]
```diff
@@ -564,6 +564,9 @@ def format_docker_cmd(docker_cmd):
         })
     # Generate ybt_bin scripts
     if ybt_bin_path:
+        if ybt_bin_path.startswith('//'):
+            ybt_bin_path = join(build_context.conf.get_bin_path(),
+                                ybt_bin_path[2::])
```
Comment (itamaro) [inline yabt/docker.py]: isn't it the same to do `[2:]`?

File: yabt/docker.py [hunk 2]
```diff
@@ -564,6 +564,9 @@ def format_docker_cmd(docker_cmd):
         })
     # Generate ybt_bin scripts
     if ybt_bin_path:
+        if ybt_bin_path.startswith('//'):
```
Comment (itamaro) [inline yabt/docker.py]: what's the use case of creating the ybt_bin under a completely different path from the related build target?

Comment (eyal-resonai) [reply yabt/docker.py]: @itamaro We are writing a DockerImage target and it seems that it doesn't generate a ybt_bin/.. script unless we use the ybt_bin_path argument (is that the expected behavior?) and currently the only path accepted as value of this arg is one which is a full path starting with the <workspace>/ybt_bin

Comment (itamaro) [reply yabt/docker.py]: > is that the expected behavior?

yes. "plain" DockerBuilder doesn't automatically know how to generate a ybt_bin (if it's even relevant) - it's the PythonApp / CppApp wrappers that bring in that knowledge.

if you follow the stack of {Python,Cpp}App builders, you'll see [this](https://github.com/resonai/ybt/blob/master/yabt/builders/dockerapp.py#L57) builds the ybt_bin_path, so you *should* be able to use something similar directly - if not, then it's indeed a bug.

File: yabt/docker.py (hunk 1 above)
Comment (eyal-resonai) [reply yabt/docker.py]: fixed

File: yabt/docker.py (hunk 2 above)
Comment (eyal-resonai) [reply yabt/docker.py]: As you said, I didn't add ybt_bin output as default but also added support for ybt_bin_path='.' as a way to have it in the default location

**Pull request merged**